.github/workflows/main.yml

```diff
@@ -173,6 +174,37 @@ jobs:
             OS-${{ runner.os }},
             VM-${{ matrix.platform }},
             Py-${{ steps.python-install.outputs.python-version }}
+            
+  check-changed-folders:
+    name: Check files
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get changed files in the _distutils folder
+        id: changed-files-specific-distutils
```

Comment:
What will this produce on non-PR triggers? Does it even need to run for those?

Reply:
Why not?

Reply:
Because this means that it'll produce a failing check for such PRs merged to the main branch.

Reply:
I am setting it to only run for PRs.